### PR TITLE
83 - front end edits to profile page

### PR DESCRIPTION
**Pull request by Jewell-Micah**

### CODE REVIEW

File: App/View/profile.php [hunk 1]
```diff
@@ -32,6 +32,26 @@
                 <input class="form-control" id="editUsername" name="editUsername" required>
                 <small id="editUsernameMessage" hidden></small> 
             </div>
+
+
+        <h2>Reset Password</h2>
```
Comment (Onik97) [inline App/View/profile.php]: Remove extra space

File: App/View/profile.php [hunk 2]
```diff
@@ -43,4 +63,11 @@
...
 </div> 
-</div>
\ No newline at end of file
+</div>
+
+<div class="link">
+    <p1>Linked With Github</p1> &nbsp <img src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png" style=”width:40px;height:20px;">  
+</div>
+<div class="link"> 
+    <p2>Linked With Discord</p2> &nbsp <img src="https://preview.redd.it/s9biyhs4lix61.jpg?auto=webp&s=c600a95eff95b3e9406a8b913c6aa3988b5e3a8b" style=”width:40px;height:20px;">
```
Comment (Onik97) [inline App/View/profile.php]: Adding Style on HTML is really bad practice, use CSS instead

File: App/View/profile.php [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <p id="navBarActive" hidden>profilePage</p>
 
-<h1>Profile Page</h1>
+<h1 class="headings">Profile Page</h1>
```
Comment (Onik97) [inline App/View/profile.php]: Use Singular on Class names

File: App/src/Style/notFound.css [hunk 4]
```diff
@@ -26,3 +26,7 @@ body {
       font-weight: bold;
       margin: 5px 0px;
   }
+
+.titles{
```
Comment (Onik97) [inline App/src/Style/notFound.css]: Change to singular

Comment (Onik97) [reply App/src/Style/notFound.css]: Plus tidy up code 
titles { 
instead of 
titles{

File: App/src/Style/profile.css [hunk 5]
```diff
@@ -1,3 +1,27 @@
...
+    border-radius: 4px;
+    box-sizing: border-box;
+}
+
+.headings {
+    position: center;
+    text-align: center;
+}
+
+.link{
```
Comment (Onik97) [inline App/src/Style/profile.css]: Clean up code 
.link {

**Review by Onik97 — CHANGES_REQUESTED** [review]

File: App/View/profile.php (hunk 2 above)
Comment (Jewell-Micah) [reply App/View/profile.php]: Not sure how to do CSS spacing so I've just removed the HTML styling and left the result as is.

**Pull request merged**